arkane/statmech.py

```diff
@@ -1395,3 +1396,125 @@ def readGjf(self):
+   tsigma2          {torsigma2}               # torsional symmetry number of hindered rotor 2
+end_torsions
+#----------------------------------#
+# Calculations                     #
+#----------------------------------#
+start_calcs
+   level            hf 3-21g        # the calculation level
+   charge           0               # charge
+   multiplicity     0               # spin multiplicity, shouldn't matter in context of Arkane
+end_calcs
+#----------------------------------#
+# Torsional PES                    #
+#----------------------------------#
+start_pes
+   t1step           10.0            # step in phi1 for scan calculation [degrees]
+   t2step           10.0            # step in phi2 for scan calculation [degrees]
+   symmetry         {symmetry}               # Symmetry condition for PES: [a,b,c,ab,ac,bc,abc] or none
+end_pes
+#----------------------------------#
+# Fitting details                  #
+#----------------------------------#
+start_fourier
+   weight           0.9             #
+   ignore           0.0             # Set to zero coefficients with smaller absolute value (in cm^-1)
+   # Fourier Terms (Even)           #
+   cos1             1-9             # i values in cos(i*Phi_1)
+   cos2             1-9             # j values in cos(j*Phi_2)
+   cos1cos2         1-7 , 1-7       # i,j values in cos(i*Phi_1) * cos(j*Phi_2)
+   sin1sin2         1-7 , 1-7       # i,j values in sin(i*Phi_1) * sin(j*Phi_2)
+   # Fourier Terms (Odd)            #
+   sin1             1-9            # i values in sin(i*Phi_1)
+   sin2             1-9            # j values in sin(j*Phi_2)
+   cos1sin2         1-7 , 1-7            # i,j values in cos(i*Phi_1) * sin(j*Phi_2)
+   sin1cos2         1-7 , 1-7            # i,j values in sin(i*Phi_1) * cos(j*Phi_2)
+end_fourier
+#----------------------------------#
+# Search and Opt stationary points #
+#----------------------------------#
+start_statpoint
+   tolerance        1.0             # step (in degrees) to explore torsional PES when looking for CPs
+   freqscal         1.000           # scaling factor for frequencies
+end_statpoint
+#----------------------------------#
+# 2D-NS Hamiltonian                #
+#----------------------------------#
+start_tor2dns
+   dijvar           yes             # yes (dij not constant) or no (dij constant)
+   kmax             100             # check 2013-JChemPhys_138_134112, eq (14)
+   maxeigen         1e4             # threshold for H eigenvalues (in cm^-1)
+end_tor2dns
+#----------------------------------#
+# Partition functions              #
+#----------------------------------#
+start_rovibpf
+   interpolation    fourier         # fourier or spline order (1,3,5)
+   integrationstep  1.0             # integration dphi
+end_rovibpf
+#----------------------------------#
+# Working temperatures             #
+#----------------------------------#
+start_temperatures                 #
+   100.0   150.0   200.0          #
```

Comment:
The temperature range is unimportant because it's only used by Q2DTor after you get the eigenvalues, I just take the eigenvalues and calculate everything ourselves once we have them.